### ISSUE TITLE
StepFunctions: ASL / state machine language LSP server crashing on 3.10.0 (but not 3.9.0)

### ISSUE THREAD

**Issue by justinmk3** [aws/aws-toolkit-vscode#5195]
_from @up-town-down in https://www.github.com/aws/aws-toolkit-vscode/issues/5151#issuecomment-2180658276 :_

> I'm seeing an error with the the state machine language server, but it's on 3.10.0 and is not occurring in 3.9.0
![Screenshot 2024-06-20 at 14 11 28](https://github.com/aws/aws-toolkit-vscode/assets/11238286/f60aa997-4e82-4644-97d6-e4fc666332ce)
log attached
[toolkit.log](https://github.com/user-attachments/files/15914430/toolkit.log)

 ```
Node.js v20.9.0
[Info  - 14:08:16] Connection to server got closed. Server will restart.
node:internal/modules/cjs/loader:1058
  throw err;
  ^

Error: Cannot find module 'vscode'
Require stack:
- /Users/user/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-3.10.0/dist/src/stepFunctions/asl/aslServer.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1055:15)
    at Module._load (node:internal/modules/cjs/loader:908:27)
    at c._load (node:electron/js2c/node_init:2:13672)
    at Module.require (node:internal/modules/cjs/loader:1122:19)
    at require (node:internal/modules/helpers:130:18)
    at /Users/user/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-3.10.0/dist/src/stepFunctions/asl/aslServer.js:1393:72630
    at /Users/user/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-3.10.0/dist/src/stepFunctions/asl/aslServer.js:1394:7362
    at Object.<anonymous> (/Users/user/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-3.10.0/dist/src/stepFunctions/asl/aslServer.js:1394:7401)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1311:10) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/user/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-3.10.0/dist/src/stepFunctions/asl/aslServer.js'
  ]
}
```

**Comment by aldoromo88** [comment]
still happening on `aws-toolkit-vscode-3.11.0`

**Comment by justinmk3** [comment]
We are working on this.

**Comment by justinmk3** [comment]
Fixed in AWS Toolkit 3.12.0. We have test coverage to prevent this issue in the future.

**Comment by up-town-down** [comment]
Confirmed - seeing this as working now - thanks @justinmk3
